WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/Edit Order Status/OrderStatusListViewController.swift

```diff
@@ -57,32 +40,22 @@ final class OrderStatusListViewController: UIViewController {
         registerTableViewCells()
         configureNavigationBar()
         configureTableView()
-
-        configureResultsController()
-
-        preselectStatusIfPossible()
+        reloadTable()
     }
 
-    /// Setup: Results Controller
-    ///
-    private func configureResultsController() {
-        statusResultsController.startForwardingEvents(to: tableView)
```

Comment:
[Q]: Are you planning to replicate this line later or was it a miss?

Reply:
Captured in https://github.com/woocommerce/woocommerce-ios/pull/3530/files#diff-fd43a78202fcd3f89a78e22d1c6dc8819ef103625f5f0b44847296528b124cf3R447 but I am reconsidering whether we really need another status results controller since the VM already could access one through the data source